perl-lib/OESS/t/Interface_test.t

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/perl  -T
+
+use OESS::Interface;
+use Test::More tests => 30;
+use Log::Log4perl;
+use OESS::DB;
+use Data::Dumper;
+use Test::Deep;
+
+
+# Initialize logging
+Log::Log4perl->init("/etc/oess/logging.conf");
```

Comment:
You can use this for testing but not for actual tests (this file does not exist when running the test suite)